src/pages/products/turn/connecting.astro

```diff
@@ -0,0 +1,181 @@
+---
+import type { IStandardPageMetadata } from '@models/products/sgid/types';
+
+import QuickLinks from '@components/page/QuickLinks.astro';
+import Section from '@components/page/Section.astro';
+import Layout from '@layouts/FullWidth.astro';
+
+import { stripeSections } from '@utils/page';
+
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+const page: IStandardPageMetadata = {
+  title: 'Getting Connected',
+  pillar: 'TURN',
+  section: stripeSections([
+    {},
+    { title: 'Getting Connected' },
+    { title: 'TURN Network' },
+    { title: 'Nevada Network' },
+    { title: 'Support Contacts' },
+  ]),
+};
+---
+
+<Layout title={page.title} pillar={page.pillar}>
+  <QuickLinks links={page.section} slot="quick-links" />
+
+  <Section {...page.section[1]}>
+    <p>
+      To create a new subscription or renew an existing subscription for Nevada GPS, you'll need to provide the
+      following information:
+    </p>
+    <ul>
+      <li>● the name of your organization (avoid punctuation);</li>
+      <li>● a login/username;</li>
+      <li>● a password (8 characters);</li>
+      <li>
+        ● a contact name, email address (once you add in your email address, it will be associated/connected with your
+        account and can't be used again), and phone number;
+      </li>
+      <li>● your physical location/address; and</li>
+      <li>● your mailing address.</li>
+    </ul>
+    <p>
+      To pay for your subscription, visit the <a href="https://turngps-billpay.ugrc.utah.gov/"
```

Comment:
**chore**: replace the `<a>` tag with an `<ExternalLink>` for anything that goes outside of gis.utah.gov.